packages/veritone-react-common/src/components/SearchBar/story.js

```diff
@@ -20,15 +26,29 @@ const transcript = {
   showPill: true
 };
 
+const recognizedText = {
```

Comment:
@atungw Why are we doing another key for each engine category?  Why not just add these attributes to `engineCategoryMapping` below and pass the engine category IDs as the `enabledEngineCategories`?

Reply:
This stuff is hardcoded for convenience, but it'll mostly be configured from here. https://enterprise.stage.veritone.com/api/engine/category?time=1517268957867